### DISCUSSION TITLE
Share the Trilium Notes on the local network

### DISCUSSION THREAD

**Discussion by Pashoupati** [TriliumNext/Trilium#4761] [Q&A] [answered]
Hello,
I have two computers on the same local network with a Samba share set up. I would like to share the data from my Trilium database on both machines. Is there a way to do this? Thank you.

**Comment by Nriver** [comment, marked as answer]
You need to deploy a Trilium server. Do not use Samba, it will break the database.

**Reply by Pashoupati** [reply]
Thanks
